MapleServer2/Servers/Game/GameSession.cs

```diff
@@ -26,9 +26,9 @@ public GameSession(FieldManagerFactory fieldManagerFactory) : base()
             FieldManagerFactory = fieldManagerFactory;
         }
 
-        public void SendNotice(string message)
+        public void SendNotice(object message)
```

Comment:
Why?

Reply:
Is better.

1. If you just one to send an object dont need to add `ToString()` or do interpolation string just for an object is easier to just pass the string or the object itself.

2.Easy to understand since you are `Sending a Notice about something`.